mixnode/src/node/node_description.rs

```diff
@@ -0,0 +1,40 @@
+use serde::Deserialize;
+use serde::Serialize;
+use std::fs::File;
+use std::path::PathBuf;
+use std::{fs, io};
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct NodeDescription {
+    pub(crate) name: String,
+    pub(crate) description: String,
+    pub(crate) link: String,
+}
+
+impl Default for NodeDescription {
+    fn default() -> Self {
+        NodeDescription {
+            name: "This node has not yet set a name".to_string(),
+            description: "This node has not yet set a description".to_string(),
+            link: "https://nymtech.net".to_string(),
+        }
+    }
+}
+
+impl NodeDescription {
+    pub(crate) fn load_from_file(mut config_path: PathBuf) -> io::Result<NodeDescription> {
+        config_path.push("descriptor.json");
+        let json = fs::read_to_string(config_path)?;
+        ::serde_json::from_str(&json)
```

Comment:
Any particular reason for the strange mixture of json and toml? I.e. we're saving node configuration in a `.toml` file but for some reason we save description as a `.json`?